Add auto agent (#80)
mbodiai/embodied-agents: README.md

- **Motor Agents** always return a `Motion`.
- **Sensory Agents** always return a `SensorReading`.

> For convenience, we also provide **AutoAgent** which dynamically initializes the right agent for the specified task. See [Deep Dive section](#auto-agent) below for more.

Reviewer: ```suggestion
For convenience, we also provide **AutoAgent** which dynamically initializes the right agent for the specified task. See [API Reference](#auto-agent) below for more.
```

- [x] OpenVLA Motor Agent
- [x] Automatic dataset recording on Robot
- [x] Yolo, SAM2, DepthAnything Sensory Agents
- [x] Auto Agent

> - [ ] Central Nervous System (CNS)

Reviewer: ```suggestion
```
Remove this

- You can `pack` a list of `Sample`s or Dicts into a single `Sample` or `Dict` and `unpack` accordingly?
- You can `unflatten` any python structure into a `Sample` class so long you provide it with a valid json schema?

> ## Deep Dive

Reviewer: ```suggestion
## API Reference
```
